src/drt/src/TritonRoute.cpp

```diff
@@ -819,6 +819,9 @@ void TritonRoute::sendDesignUpdates(const std::string& globals_path)
 
 int TritonRoute::main()
 {
+  if (DBPROCESSNODE == "GF14_13M_3Mx_2Cx_4Kx_2Hx_2Gx_LB") {
+    USENONPREFTRACKS = false;
+  }
```

Comment:
This is in TritonRoute::initGuide already.  Are you needing to move it earlier?  If so the other can be removed.